src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -1,29 +1,16 @@
-import functools
 import logging
 
 from arroyo.types import Message
 
+from sentry.sentry_metrics import indexer
 from sentry.sentry_metrics.configuration import UseCaseKey
 from sentry.sentry_metrics.consumers.indexer.batch import IndexerBatch
 from sentry.sentry_metrics.consumers.indexer.common import MessageBatch
+from sentry.utils import metrics
 
 logger = logging.getLogger(__name__)
 
 
-@functools.lru_cache(maxsize=10)
```

Comment:
I got rid of those because otherwise the return type of `indexer` is `Any` in `process_messages`, and that would infect return values of `indexer.bulk_record`, and `get_fetch_metadata` (where i shipped a bug/crash last time)